README.md

```diff
@@ -1,25 +1,20 @@
 ## README ##
 
-This is a simple grant application management service for Greater Poweshiek Community Foundation, deployed at 
+This is a simple grant application management service for Greater Poweshiek Community Foundation, deployed at
 https://grant-app.herokuapp.com/
 
 **Repository Properties**
 
-- Ruby version: 1.9.3
-- Rails version: 4.2.0 
+- Ruby version: 2.3.0
+- Rails version: 4.2.0
 
 **Cloning and Configuration Instructions**
 - Clone the repository at https://github.com/CSC322-Grinnell/grantapp
     - _It is advised that you create a development branch for easier version control_
-- Use Ruby's gem bundler with the command-line instruction 
-    ``` bundle install ```
-- Create the database for the current Rails environment 
-    ``` rake db:create ```
-- Run migrations for the current environment that have not run yet. 
-    _By default it will run migrations only in the development environment._
-    ``` rake db:migrate ```
-- At this point you should be able to run the website on the local rails server. 
-    ``` rails server -b $IP -p $PORT ```
+- Set up the application
+    ``` bin/setup ```
```

Comment:
💁 Instead of having multiple commands for new team members to run when setting up your application, you can add all of your setup commands to a setup script. Rails comes with an example setup script, so I updated it with the commands in the README.